src/main/java/uk/gov/ch/model/update/EntityData.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.ch.model.update;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+public class EntityData {
+
+    @Id
+    @Column(name = "corporate_body_id")
+    @JsonIgnore
+    private Long corporateBodyId;
+
+    @Column(name = "incorporation_number")
```

Comment:
don't think we need to pass back the incorporation_number

Reply:
No worries. We copied your SQL from our Teams chat Ad Verbatim, so weren't sure if it was going to be used by the client for some reason.